docs/source/release_notes.rst

```diff
@@ -11,6 +11,7 @@ Future Release
         * The criteria for categorical type inference have changed (:pr:`1065`)
         * The meaning of both the ``categorical_threshold`` and
           ``numeric_categorical_threshold`` settings have changed (:pr:`1065`)
+        * Make sampling for type inference more consistent (:pr:`1083`)
```

Comment:
would the change in sampling size be considered a breaking change because the logical types inferred at init could be different now than they were before for the same dataframe (both because we're now using the head and also taking more entries)?

Reply:
Yeah, I suppose it could be.  I'm not sure actually.  You'll potentially get a different result but you'll also probably get a more correct one.  I suppose it's worth a note at least.

Reply:
Recommended update made here: https://github.com/alteryx/woodwork/pull/1083/commits/3b6661b5f4c33b09e05022cddd2c7010d0b700a4

Reply:
Actually I made some edits to improve the wording.  Probably should just look at the overall changes tab.

Reply:
breaking changes section looks good to go!